CPAC/__main__.py

```diff
@@ -313,10 +313,15 @@ def accept_all(object, name, value):
         from nipype.scripts.crash_files import display_crash_file
         display_crash_file(crash_file, False, False, None)
 
+
 @utils.group()
-def data_config():
-    from CPAC.utils.ga import track_config
-    track_config('cli')
+@click.option('--tracking_opt-out', is_flag=True,
```

Comment:
To make sure I understand, are we using `click` and `argparse`/`optparse`? Is there a reason for using both?

Reply:
Not after https://github.com/FCP-INDI/C-PAC/pull/1716/commits/934d3511fd1f89f7348b0baaeb2ae3fbbe66b36c!